libstuff/libstuff.cpp

```diff
@@ -2614,12 +2630,10 @@ int SQuery(sqlite3* db, const char* e, const string& sql, SQResult& result, int6
     }
 
     uint64_t elapsed = STimeNow() - startTime;
-    string sqlToLog = sql;
```

Comment:
This copied the entire query string on every query, including successful ones, for the purpose of logging failures. This is expensive for 100mb+ queries, and waasteful even for smaller ones. I've moved this to only happen when a failure actually occurs (see below in a couple locations).